docs/developing.md

```diff
@@ -39,6 +39,7 @@
 ### External tools
 
 - [kustomize](https://github.com/kubernetes-sigs/kustomize#kustomize)
+- [kind](https://github.com/kubernetes-sigs/kind) (0.5.1)
```

Comment:
I would rather see the version limitation spelled out explicitly in the `Deploying with Kubernetes In Docker` section. The kind tool is specific to that section.

Reply:
In other words, I do not think that kind should be added to the `External Tools`.

Reply:
Done